pkg/synthesis/k8sPolicies.go

```diff
@@ -0,0 +1,135 @@
+package synthesis
+
+import (
+	"fmt"
+
+	networking "k8s.io/api/networking/v1"
+	meta "k8s.io/apimachinery/pkg/apis/meta/v1"
+	admin "sigs.k8s.io/network-policy-api/apis/v1alpha1"
+
+	"github.com/np-guard/vmware-analyzer/pkg/model/dfw"
+	"github.com/np-guard/vmware-analyzer/pkg/symbolicexpr"
+)
+
+var abstractToAdminRuleAction = map[dfw.RuleAction]admin.AdminNetworkPolicyRuleAction{
+	dfw.ActionAllow:     admin.AdminNetworkPolicyRuleActionAllow,
+	dfw.ActionDeny:      admin.AdminNetworkPolicyRuleActionDeny,
+	dfw.ActionJumpToApp: admin.AdminNetworkPolicyRuleActionPass,
+}
+
+type k8sPolicies struct {
+	networkPolicies      []*networking.NetworkPolicy
+	adminNetworkPolicies []*admin.AdminNetworkPolicy
+}
+
+func (policies *k8sPolicies) toNetworkPolicies(model *AbstractModelSyn) ([]*networking.NetworkPolicy, []*admin.AdminNetworkPolicy) {
+	for _, p := range model.policy {
+		policies.symbolicRulesToPolicies(model, p.outbound, false)
+		policies.symbolicRulesToPolicies(model, p.inbound, true)
+	}
+	return policies.networkPolicies, policies.adminNetworkPolicies
+}
+
+func (policies *k8sPolicies) symbolicRulesToPolicies(model *AbstractModelSyn, rules []*symbolicRule, inbound bool) {
+	for _, rule := range rules {
+		isAdmin := model.allowOnlyFromCategory > rule.origRuleCategory
+		paths := &rule.allowOnlyRulePaths
+		if isAdmin {
+			paths = rule.origSymbolicPaths
+		}
+		for _, p := range *paths {
+			if !p.Conn.TCPUDPSet().IsEmpty() {
+				policies.addNewPolicy(p, inbound, isAdmin, rule.origRule.Action)
+			}
```

Comment:
please add `else` that prints to `logging.Debugf` info about ignored rules

Reply:
done